CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ## [Unreleased]
 ### Fixed
+- Add `package.json` to node module `exports`
```

Comment:
To be clearer I'd maybe say "Ensure `package.json` can be imported by consuming projects"